### ISSUE TITLE
AssertionError When Creating Wallet - Python 2.7

### ISSUE THREAD

**Issue by bobozar** [blockcypher/blockcypher-python#61]
>>> create_hd_wallet(wallet_name='test', 
 xpubkey='xpub661MyMwAqRbcFtXgS5sYJABqqG9YLmC4Q1Rdap9gSE8NqtwybGhePY2gZ29ESFjqJoCu1Rupje8YtGqsefD265TMg7usUDFdp6W1EGMcet8', api_key=token)

```
Traceback (most recent call last):
  File "<pyshell#3>", line 1, in <module>
    create_hd_wallet(wallet_name='test', xpubkey='xpub661MyMwAqRbcFtXgS5sYJABqqG9YLmC4Q1Rdap9gSE8NqtwybGhePY2gZ29ESFjqJoCu1Rupje8YtGqsefD265TMg7usUDFdp6W1EGMcet8', api_key=token)
  File "C:\Python27\lib\site-packages\blockcypher\api.py", line 1336, in create_hd_wallet
    assert inferred_coin_symbol == coin_symbol
AssertionError
```

**Comment by Latand** [comment]
I had the same issue! Dash and BTC have the same first 4 letters of extended public key, starts from "xpub", so if you try to add BTC wallet  - the assertion violates because it looks for "dash" as coin_symbol.
Can`t beleive this error still exists since 31 dec

**Comment by busbyjon** [comment]
Still seeing this error crop up...

**Comment by quentinlesceller** [comment]
Fixed by #77.
